### PR TITLE
Add toggle component

### PR DESCRIPTION
**Pull request by Prokyonn**
Adds a component to easily toggle classes on click events

### CODE REVIEW

File: packages/components/toggle/toggle.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     */
+    toggle.getFirstClass = function getFirstClass(element) {
+        return element.getAttribute('class').split(' ')[0];
+    };
+
+    toggle.bindEvents = function bindEvents() {
+        toggle.el.addEventListener('click', toggle.toggle);
+    };
+
+    toggle.toggle = function toggle() {
```
Comment (alexander-schranz) [inline packages/components/toggle/toggle.js:43]: atleast firefox don't like that the method is called `toggle` here it throws an error `toggle.el` is undefined so we need to rename this method.

Comment (alexander-schranz) [reply packages/components/toggle/toggle.js:43]: Example fiddle: https://jsfiddle.net/8jyamdtz/

File: packages/components/toggle/toggle.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     * import Toggle from '@sulu/web/packages/components/toggle/toggle';
+     * var component = new Toggle();
+     * component.initialize(document.getElementById('toggle'), {});
+     *
+     * @param {HTMLElement} el
+     * @param {object} options
+     */
+    toggle.initialize = function initialize(el, options) {
+        toggle.el = el;
+        toggle.id = el.id;
```
Comment (alexander-schranz) [inline packages/components/toggle/toggle.js:23]: Not required can be removed:
```suggestion
```

File: packages/components/toggle/toggle.js (hunk 1 above)
Comment (Prokyonn) [reply packages/components/toggle/toggle.js:42]: Updated it: https://jsfiddle.net/74h1xsaL/ works for me in safari/chrome/firefox

**Review by alexander-schranz — APPROVED** [review]

File: packages/components/toggle/toggle.js (hunk 1 above)
Comment (Prokyonn) [reply packages/components/toggle/toggle.js]: just forgot to push 😬

**Pull request merged**